wsgi/playdoh.wsgi

```diff
@@ -9,7 +11,7 @@ except ImportError:
 
 
 if newrelic:
-    newrelic_ini = os.getenv('NEWRELIC_PYTHON_INI_FILE', False)
+    newrelic_ini = config('NEWRELIC_PYTHON_INI_FILE', default='newrelic.ini')
```

Comment:
Agree that changing default can cause errors in this double deployment scenario we're in. I changed it to default to `False` again and updated the `bedrock-dev` apps with the needed env variables.